frontend/src/services/http/http.service.ts

```diff
@@ -20,10 +31,13 @@ class Http {
       .catch(this.throwError);
   }
 
-  private getHeaders(): Headers {
+  private getHeaders(hasAuth?: boolean): Headers {
     const headers = new Headers();
 
-    headers.append(HttpHeader.CONTENT_TYPE, 'application/json');
```

Comment:
For sign-in, sign-up we need to add content type, you need to add it to http options and pass for sign-in/sign-up, and then accept here in params